### ISSUE TITLE
Framework provides consistent interpretation

### ISSUE THREAD

**Issue by moekraft** [w3c/wcag-act#71]
From Survey, https://www.w3.org/2002/09/wbs/93339/ACTFrameworkFPWD/results

David McDonald suggests, 
This paragraph is overly strong and not necessary, and undermines the credibility of WCAG.

"Accessibility experts often disagree on how accessibility requirements should be tested. These disagreements lead to conflicting results of accessibility tests. This is true for both manual accessibility tests as well as for automated testing done through accessibility test tools (ATTs)."

Suggest;
"This framework is intended to provide a central point where accessibility experts can agree on how accessibility requirements should be tested so as to avoid conflicting results of accessibility tests. It is intended for both manual accessibility tests as well as for automated testing done through accessibility test tools (ATTs)."

From 3/15/17 meeting, Proposed:
_This framework is intended to provide a consistent interpretation of how to test for accessibility requirements so as to avoid conflicting results of accessibility tests. It is intended for both manual accessibility tests as well as for automated testing done through accessibility test tools (ATTs)_.

+1 Shadi
+1 Wilco
+1 Charu

**Comment by moekraft** [comment]
Pull request #72 ready for survey.
